<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>410</y>
     <width>94</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>登 录</string>
   </property>
  </widget>
  <widget class="QWidget" name="iconBk" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>21</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/login/images/coffeechat.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>61</width>
     <height>37</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>飞鸽</string>
   </property>
  </widget>
  <widget class="QLabel" name="labError">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>379</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 59, 48);</string>
   </property>
   <property name="text">
    <string>timeout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>21</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/login/images/btn-setting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserName">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>255</y>
     <width>295</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:5px;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>xmcy0011@sina.com</string>
   </property>
   <property name="placeholderText">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePwd">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>320</y>
     <width>295</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-left:5px;</string>
   </property>
   <property name="text">
    <string>sa123456</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>450</y>
     <width>94</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 122, 255);</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
